Subject: mac80211: allow WDS mode

This allows creating interfaces in WDS mode or switching
existing ones into WDS mode (both via cfg80211.)

Signed-off-by: Johannes Berg <johannes@sipsolutions.net>

---
 net/mac80211/cfg.c |    2 ++
 1 file changed, 2 insertions(+)

--- everything.orig/net/mac80211/cfg.c	2007-11-07 13:19:27.981515569 +0100
+++ everything/net/mac80211/cfg.c	2007-11-07 13:19:29.441515732 +0100
@@ -29,6 +29,8 @@ nl80211_type_to_mac80211_type(enum nl802
 		return IEEE80211_IF_TYPE_AP;
 	case NL80211_IFTYPE_AP_VLAN:
 		return IEEE80211_IF_TYPE_VLAN;
+	case NL80211_IFTYPE_WDS:
+		return IEEE80211_IF_TYPE_WDS;
 	default:
 		return IEEE80211_IF_TYPE_INVALID;
 	}
